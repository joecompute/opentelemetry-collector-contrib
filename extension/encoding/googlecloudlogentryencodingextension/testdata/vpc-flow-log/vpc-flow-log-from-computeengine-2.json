{
  "insertId": "qev5urf2og0oq",
  "jsonPayload": {
    "bytes_sent": "0",
    "end_time": "2025-09-27T21:36:34.265718080Z",
    "connection": {
      "dest_ip": "10.128.0.43",
      "dest_port": 443,
      "src_port": 47036,
      "src_ip": "35.191.240.44",
      "protocol": 6
    },
    "reporter": "DEST",
    "start_time": "2025-09-27T21:36:34.265718080Z",
    "dest_vpc": {
      "subnetwork_name": "default",
      "vpc_name": "default",
      "subnetwork_region": "us-central1",
      "project_id": "elastic-obs-integrations-dev"
    },
    "network_service": {
      "dscp": 32
    },
    "dest_instance": {
      "vm_name": "proxy-crest-new-group-892l",
      "region": "us-central1",
      "zone": "us-central1-a",
      "managed_instance_group": {
        "zone": "us-central1-a",
        "name": "proxy-crest-new-group"
      },
      "project_id": "elastic-obs-integrations-dev"
    },
    "packets_sent": "16"
  },
  "resource": {
    "type": "gce_subnetwork",
    "labels": {
      "subnetwork_id": "2319928369511802306",
      "project_id": "elastic-obs-integrations-dev",
      "location": "us-central1",
      "subnetwork_name": "default"
    }
  },
  "timestamp": "2025-09-27T21:36:40.414372716Z",
  "logName": "projects/elastic-obs-integrations-dev/logs/compute.googleapis.com%2Fvpc_flows",
  "receiveTimestamp": "2025-09-27T21:36:40.414372716Z"
}