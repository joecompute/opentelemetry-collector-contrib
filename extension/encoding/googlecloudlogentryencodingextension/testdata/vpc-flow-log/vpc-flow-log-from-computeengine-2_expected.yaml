resourceLogs:
  - resource:
      attributes:
        - key: gcp.project
          value:
            stringValue: elastic-obs-integrations-dev
        - key: cloud.resource_id
          value:
            stringValue: compute.googleapis.com%2Fvpc_flows
        - key: gcp.resource_type
          value:
            stringValue: gce_subnetwork
        - key: gcp.label.subnetwork_id
          value:
            stringValue: "2319928369511802306"
        - key: gcp.label.project_id
          value:
            stringValue: elastic-obs-integrations-dev
        - key: gcp.label.location
          value:
            stringValue: us-central1
        - key: gcp.label.subnetwork_name
          value:
            stringValue: default
    scopeLogs:
      - logRecords:
          - attributes:
              - key: log.record.uid
                value:
                  stringValue: qev5urf2og0oq
              - key: network.transport
                value:
                  stringValue: tcp
              - key: source.address
                value:
                  stringValue: 35.191.240.44
              - key: destination.address
                value:
                  stringValue: 10.128.0.43
              - key: source.port
                value:
                  intValue: "47036"
              - key: destination.port
                value:
                  intValue: "443"
              - key: gcp.vpc.flow.reporter
                value:
                  stringValue: DEST
              - key: gcp.vpc.flow.bytes_sent
                value:
                  intValue: "0"
              - key: gcp.vpc.flow.packets_sent
                value:
                  intValue: "16"
              - key: gcp.vpc.flow.start_time
                value:
                  stringValue: "2025-09-27T21:36:34.265718080Z"
              - key: gcp.vpc.flow.end_time
                value:
                  stringValue: "2025-09-27T21:36:34.265718080Z"
              - key: gcp.vpc.flow.destination.instance.project.id
                value:
                  stringValue: elastic-obs-integrations-dev
              - key: gcp.vpc.flow.destination.instance.vm.region
                value:
                  stringValue: us-central1
              - key: gcp.vpc.flow.destination.instance.vm.name
                value:
                  stringValue: proxy-crest-new-group-892l
              - key: gcp.vpc.flow.destination.instance.vm.zone
                value:
                  stringValue: us-central1-a
              - key: gcp.vpc.flow.destination.instance.managed_instance_group.name
                value:
                  stringValue: proxy-crest-new-group
              - key: gcp.vpc.flow.destination.instance.managed_instance_group.zone
                value:
                  stringValue: us-central1-a
              - key: gcp.vpc.flow.destination.project.id
                value:
                  stringValue: elastic-obs-integrations-dev
              - key: gcp.vpc.flow.destination.subnet.name
                value:
                  stringValue: default
              - key: gcp.vpc.flow.destination.subnet.region
                value:
                  stringValue: us-central1
              - key: gcp.vpc.flow.destination.vpc.name
                value:
                  stringValue: default
              - key: gcp.vpc.flow.network_service.dscp
                value:
                  intValue: "32"
            body: {}
            observedTimeUnixNano: "1727500600414372716"
            timeUnixNano: "1727500600414372716"
        scope: {}
